modules/@apostrophecms/schema/ui/apos/components/AposAttachment.vue

```diff
@@ -50,19 +56,78 @@ export default {
     },
     disabled () {
       return (this.limitReached || this.field.disabled);
+    },
+    moduleOptions() {
+      return window.apos.modules['@apostrophecms/file'];
     }
   },
   methods: {
     updated(items) {
       this.next = items;
     },
     validate(value) {
-      if (this.field.required && !value.data.length) {
+      if (this.field.required && !value) {
         return 'required';
       }
 
       return false;
-    }
+    },
+    async uploadMedia (event) {
+      this.$emit('upload-started');
+      const file = event.target.files[0];
+
+      const emptyDoc = await apos.http.post(this.moduleOptions.action, {
+        body: {
+          _newInstance: true
+        }
+      });
+      await apos.notify('Uploading file', { dismiss: true });
```

Comment:
'Uploading file' becomes 'Uploading FILENAME.EXT' 
for example 'Uploading DJT_2014taxes.pdf' 
Filenames should have a max char length of 35 before truncating to an ellipse 
for example 'Uploading big_big_long_title-As-far-as-estimate....pdf'

Reply:
Also, use this cloud upload logo.. I don't think we've used it before so it will have to be globally imported
![image](https://user-images.githubusercontent.com/1889830/94756724-4ee35180-0366-11eb-953c-334ff340cfde.png)